src/MusicalScore/Graphical/VexFlow/VexFlowStaffEntry.ts

```diff
@@ -29,6 +30,12 @@ export class VexFlowStaffEntry extends GraphicalStaffEntry {
                 gve.vfStaveNote.setStave(stave);
                 gve.applyBordersFromVexflow();
                 this.PositionAndShape.RelativePosition.x = gve.vfStaveNote.getBoundingBox().x / unitInPixels;
+                const sourceNote: Note = gve.notes[0].sourceNote;
+                if (sourceNote.isRest() && sourceNote.Length.WholeValue === 1) { // whole rest
+                    this.PositionAndShape.RelativePosition.x -= 2.6; // xShift from VexFlowConverter
```

Comment:
Yep we should create an EngravingRule for this.

Reply:
But this is not really an Engraving issue, it's just that Vexflow returns a nonsense bounding box. If we create an EngravingRule, we might need to change the RelativePosition shift and the voice entry borders.
It took me a lot of trial and error to get decent values for RelativePosition.x and voice entry borders, so i hope it won't be changed so soon.